_config.yml

```diff
@@ -19,11 +19,11 @@ exclude:
   # DO NOT comment back in, we do not want 404s
   # You can remove them from the menu by editing
   # conference.site_navigation, below
-  # - lead-a-session.md
```

Comment:
I don't understand this commit.  You seem to have *excluded* `lead-a-session.md` from the build.